README.md

```diff
@@ -27,4 +27,4 @@ Follow the next requirements to pass the tests:
 - Install the Prettier Extension and use these [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guidelines](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_tabs-js/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://AlexandeeerG.github.io/react_tabs-js/) and add it to the PR description.
```

Comment:
Add the demo link to the PR description and remove all these text 
<img width="911" alt="image" src="https://github.com/user-attachments/assets/15e63569-d8f2-4f81-9804-c29c55355363">